strawberry/permission.py

```diff
@@ -158,9 +158,14 @@ def __init__(
     def apply(self, field: StrawberryField) -> None:
         """Applies all of the permission directives to the schema and sets up silent permissions."""
         if self.use_directives:
-            field.directives.extend(
+            # Dedupe multiple directives
+            # https://github.com/strawberry-graphql/strawberry/issues/3596
```

Comment:
I don't think we need the issue linked here as it is easy to check the commit that changed this on git history

Reply:
yep, ok